onnxruntime/python/tools/transformers/models/llama/README.md

```diff
@@ -372,3 +372,37 @@ python3 -m models.llama.benchmark_all \
     --num-runs 1000 \
     --timeout 60  # number of minutes before moving to the next benchmark
 ```
+
+# Mistral
+
+## Introduction
+
+These tools for LLAMA also allow the quantization and optimizaton of Mistral in ORT. 
+
+## Exporting Mistral
+
+There is currently one supported way to export Mistral to ONNX format:
+
```

Comment:
Can Mistral be exported with the `convert_to_onnx` tool?

Reply:
The last time I tried, I hit some issues with the torchscript exporter and had to fall back to optimum. We could incorporate the optimum export into the `convert_to_onnx` tool, but this PR needs to be merged soon for the AI dev conference so it might be best suited as another PR.